DEVELOPING.md

```diff
@@ -20,7 +20,7 @@ The `stable` branch represents the current stable sbt release. Only bug fixes ar
 
 ### Note on supported JDK version for the SBT build
```

Comment:
I just noticed all cap SBT here. We typically stylize it with all lowercase like it's a command-line program. Could lowercase to sbt while you're at it plz?